rfc6962-bis.xml

```diff
@@ -588,11 +587,31 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
       <section title="Merkle Tree Head" anchor="tree_head">
         <figure>
           <preamble>
-            The log stores information about its Merkle Tree in a <spanx style="verb">TransItem</spanx> structure of type <spanx style="verb">tree_head_v2</spanx>, which encapsulates a <spanx style="verb">TreeHeadDataV2</spanx> structure:
+            The log may store information about its Merkle Tree in a <spanx style="verb">TreeHeadDataV2</spanx>:
```

Comment:
Why "stores -> may store"?  The log has to produce a TreeHeadDataV2 structure in order to produce an STH, so it has to "store" the TreeHeadDataV2 structure at least temporarily.

I think keep this as "stores".